resources/js/components/Session.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+          <div class="modal-content">
+              <div class="modal-header">
+                  <h5 class="modal-title">{{ title }}</h5>
+              </div>
+              
+              <div class="modal-body mb-1">
+                <span v-html="message"></span>
+                 <div class="progress">
+                <div class="progress-bar progress-bar-striped" role="progressbar" :style="{width: time + '%'}"><span align="left" class="pl-2">{{moment().startOf('day').seconds(time).format('mm:ss')}}</span></div>
+              </div>
+              </div>
+
+              <div class="modal-footer">
+                  <a role="button" class="btn btn-outline-secondary ml-2" href="/logout" :disabled="disabled">{{('LogOut')}}</a>
```

Comment:
Why are these disabled?

Reply:
They are only disabled when they have been clicked once so that they cant click multiple times